redcap_bridge/utils.py

```diff
@@ -58,4 +61,37 @@ def get_repo_state(path):
         commit_hash = ''
         clean = False
 
-    return commit_hash, clean
\ No newline at end of file
+    return commit_hash, clean
+
+
+def compressed_record(csv_file, compressed_file=None):
+    custom_csv = pd.read_csv(csv_file)
+    df = pd.DataFrame(custom_csv)
+    compressed_file = df.filter(regex='.___.')
+    compressed_file.replace(0,'', inplace=True)
+    for column in compressed_file:
+        if 1 in compressed_file[column].values:
+            style_moda = re.search('___(.+?)', column).group(1)
+            compressed_file[column].replace(1, style_moda, inplace=True)
+        else:
+            compressed_file.pop(column)
```

Comment:
When removing '0' columns here, doesn't this mean you are not listing modalities at the end if no choice was selected for them, i.e. all modality columns only contained '0'?